modules/tensor_mechanics/test/tests/CylindricalRankTwoAux/tests

```diff
@@ -1,13 +1,22 @@
 [Tests]
+  issues = '#716'
+  design = 'syntax/Kernels/TensorMechanics/index.md source/auxkernels/CylindricalRankTwoAux.md'
   [./test]
     type = 'Exodiff'
     input = 'test.i'
     exodiff = 'test_out.e'
+    rel_err = 2e-3
```

Comment:
This is also pretty loose.  Exodiff file?

Reply:
Sure. I'll do that for both of these.